setup.cfg

```diff
@@ -66,6 +66,7 @@ console_scripts =
 [options.extras_require]
 benchmark = py-cpuinfo==9.0.0
     mdutils==1.4.0
+    natsort==8.2.0
```

Comment:
We should avoid adding any new dependencies if we can. I think we can use Python's native sorting functionality for this.

Reply:
I agree.